tests/acceptance/wcag/pagenavigator.js

```diff
@@ -0,0 +1,31 @@
+const { waitTillHTMLRendered, getQueryParamsString } = require('./utils');
+
+/**
+ * Responsible for navigating pages
+ */
+class PageNavigator {
```

Comment:
Can/should we use this `PageNavigator` for our Percy snapshots as well? Right now, we manually navigate between pages when getting the snapshots.